staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go

```diff
@@ -139,3 +185,13 @@ func UpdateLeaseObjectCount(count int64) {
 	// See pkg/storage/etcd3/lease_manager.go
 	etcdLeaseObjectCounts.WithLabelValues().Observe(float64(count))
 }
+
+// RecordListEtcd3Metrics notes various metrics of the cost to serve a LIST request
+func RecordListEtcd3Metrics(resource string, numQueries, numFetched, predicateComplexity, numEvald, numReturned int) {
```

Comment:
Since the metrics were renamed to storage metrics, it would be great to also update the name is the function to align with the other changes.